scripts/generate_bindings.sh

```diff
@@ -43,6 +43,7 @@ bindgen $common_params \
   -- \
   -I"$PLAYDATE_C_API" \
   -I"$(which arm-none-eabi-gcc)/../arm-none-eabi/include" \
+  -I"$($(which arm-none-eabi-gcc) -print-sysroot)/include" \
```

Comment:
```suggestion
  -I"$(arm-none-eabi-gcc -print-sysroot)/include" \
```

Just this should work fine for me! I keep forgetting `-print-sysroot` exists

You'll find this funny but `-print-sysroot` actually gives `/nix/store/dqr6x4jcgd2cqksymsqvgvhb6m4ksnra-gcc-arm-embedded-12.2.rel1/bin/../arm-none-eabi` for me 😆